server/go/server.go

```diff
@@ -97,6 +98,33 @@ func handlePublicKey(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func handleGetCheckoutSession(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		http.Error(w, http.StatusText(http.StatusMethodNotAllowed), http.StatusMethodNotAllowed)
+		return
+	}
+
+	keys, ok := r.URL.Query()["sessionId"]
+
+    if !ok || len(keys[0]) < 1 {
+        log.Println("CheckoutSession ID is missing from URL", r.URL.Query())
+        return
+    }
```

Comment:
the formatting looks like this file wasn’t run through `gofmt`.

Reply:
What's the formatting bug here? Ran it through gofmt and didn't see a change

Reply:
Nevermind, I see it 👍